### PR TITLE
Provider Form

### LINKED ISSUE #53: Integrate Provider form
required by #52

### PR DESCRIPTION
**Pull request by Christiaanvdm**
This closes #53 

NewGIF

Please note:
- The name of the provider is being passed as a constant from the view.
- The user is redirected to the home page currently.

### CODE REVIEW

File: django_project/core/base_templates/index.html [hunk 1]
```diff
@@ -19,15 +19,15 @@
...
+	<link rel="stylesheet" href="https://cdn.rawgit.com/openlayers/openlayers.github.io/master/en/v5.3.0/css/ol.css" type="text/css">
 
 	<script src="https://code.jquery.com/jquery-3.3.1.slim.min.js"
 			integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo"
 			crossorigin="anonymous"></script>
 	<script src="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.bundle.min.js"
 			integrity="sha384-xrRywqdh3PHs8keKZN+8zzc5TX0GRTLCcmivcbNJWm2rs5C8PRhcEn3czEjhAO9o"
 			crossorigin="anonymous"></script>
-
+	<script src="https://cdn.rawgit.com/openlayers/openlayers.github.io/master/en/v5.3.0/build/ol.js"></script>
```
Comment (pierrealixt) [inline django_project/core/base_templates/index.html:31]: Why do we need OpenLayers?

File: django_project/ford3/templates/provider_form.html [hunk 2]
```diff
@@ -0,0 +1,292 @@
...
+				<div class="col-3">
+					<button class="edu-button edu-button-blue">Submit</button>
+				</div>
+			</div>
+			<div class="row mt1">
+			</div>
+		</div>
+	</div>
+
+	<style>
```
Comment (pierrealixt) [inline django_project/ford3/templates/provider_form.html]: maybe put this style into a css file?

File: django_project/ford3/templates/provider_form.html [hunk 3]
```diff
@@ -0,0 +1,292 @@
...
+		.campus-name-fade-in {
+			-webkit-animation: fadein 2s; /* Safari, Chrome and Opera > 12.1 */
+			-moz-animation: fadein 2s; /* Firefox < 16 */
+			-ms-animation: fadein 2s; /* Internet Explorer */
+			-o-animation: fadein 2s; /* Opera < 12.1 */
+			animation: fadein 2s;
+		}
+	</style>
+
+	<script type="text/javascript">
```
Comment (pierrealixt) [inline django_project/ford3/templates/provider_form.html]: maybe put this javascript code into a js file?

File: django_project/core/base_templates/index.html (hunk 1 above)
Comment (Christiaanvdm) [reply django_project/core/base_templates/index.html:31]: The user needs to confirm their location on the map. I created a ticket to implement this completely (I want to isolate this job) https://github.com/kartoza/ford3/issues/100

File: django_project/ford3/tests/functional/test_homepage.py [hunk 4]
```diff
@@ -1,17 +1,17 @@
-# coding=utf-8
-import unittest
+from django.test import TestCase
+from selenium import webdriver
+from selenium.webdriver.common.desired_capabilities import DesiredCapabilities
 
-from ford3.tests.functional.utils import SeleniumTestCase, selenium_flag_ready
 
+class TestHomepage(TestCase):
```
Comment (pierrealixt) [inline django_project/ford3/tests/functional/test_homepage.py]: no you should use what Rizky did. Your Test class must inherits from SeleniumTestCase.

File: django_project/ford3/tests/functional/test_provider_functionality.py [hunk 5]
```diff
@@ -0,0 +1,111 @@
+
+from selenium import webdriver
+from selenium.webdriver.common.desired_capabilities import DesiredCapabilities # noqa
+from django.test import TestCase
+
+
+class TestProviderForm(TestCase):
```
Comment (pierrealixt) [inline django_project/ford3/tests/functional/test_provider_functionality.py]: same here

**Review by pierrealixt — APPROVED** [review]
LGTM

**Pull request merged**